contentcuration/contentcuration/frontend/channelEdit/components/Clipboard/ContentNode.vue

```diff
@@ -137,13 +141,69 @@
         }
         return {};
       },
+      shouldPreload() {
+        return this.contentNode.total_count > 0;
+      },
+      loadClipboardNodesPayload() {
+        return { parent: this.nodeId, ancestorId: this.childAncestorId };
+      },
+    },
+    watch: {
+      open(open) {
+        if (!this.shouldPreload) {
+          return;
+        }
+
+        if (open) {
+          // If not loaded by the time the user clicks on this topic to open it,
+          // then we'll trigger a load immediately
+          if (!this.preloaded) {
+            this.preloaded = true;
+            this.cancelPreload();
+            this.loadClipboardNodes(this.loadClipboardNodesPayload);
```

Comment:
Note that some of the on the fly node annotation that I introduced in #2516 relies on this data having been loaded. Did adding this logic cause any issues? The main place you would see it is in descendant counts for topics with resources that have been deleted inside them.

Maybe we should also revisit the strategy of loading within each individual component, and have a way of making fewer requests for the desired data?

Reply:
@rtibbles I just tested deleting nodes from the clipboard and I didn't notice any regressions. I did find a lingering issue deleting a topic within a topic. When I delete a nested topic, it re-appears almost immediately. This is present on develop though.

The reason I did this was because the lazy preload logic I added would cause too many simultaneous requests. For instance, if you have copied a topic with several children that are also topics, when you open the parent topic all of the children topics will be mounted, which trigger a preload of their children.